swan-lake/learn/quick-tour.md

```diff
@@ -56,122 +63,71 @@ You get the following output.
 [ballerina/http] started HTTP/WS listener 0.0.0.0:9090
 ```
 
-This means your service is up and running. You can invoke the service using an HTTP client. In this case, we use cURL.
+This means your service is up and running. 
+
+> **Note:** You can test the service by invoking it using an already-available HTTP client. For example, execute the command below in a new CLI tab to use [cURL](https://curl.haxx.se/download.html) as the client.
 
 ```bash
-$ curl http://localhost:9090/hello/sayHello
+curl http://localhost:9090/hello/sayHello
 ```
 
-> **Tip**: If you do not have cURL installed, you can download it from [https://curl.haxx.se/download.html](https://curl.haxx.se/download.html).
-
 You get the following response.
 
-```
+```bash
 Hello Ballerina!
 ```
 
-Alternatively, you can use a Ballerina HTTP client to invoke the service.
+Alternatively, you can create a Ballerina HTTP client and use that to invoke the service as follows.
 
-## Using a Client to Interact with a Network-Accessible Service
+### Creating a 'helloClient' to Invoke the 'hello' Service
```

Comment:
We don't usually call something a "helloClient", similar to hello world programs or a hello service. We can just say "Creating an HTTP Client to...".

Reply:
Done.